### ISSUE TITLE
OBE: *_controller check arguments

### ISSUE THREAD

**Issue by ahelsing** [GENI-NSF/geni-portal#203]
The various *_controller functions need to check arguments before doing DB queries.

EG, a missing project ID to lookup_project gives a cryptic error like:
```

DB ERROR: : 'Array', details: '_doQuery: [Error message: Could not execute statement]\n[Last executed query: select project_name, lead_id, project_email, project_purpose FROM pa_project WHERE project_id = _]\n[Native message: ERROR:  invalid input syntax for uuid: ""\nLINE 1: ...email, project_purpose FROM pa_project WHERE project_id = ''\n                                                                     ^]\n', doing query: 'select project_name, lead_id, project_email, project_purpose FROM pa_project WHERE project_id = '_<br/>\n
```

 _Imported from trac ticket [#6](http://trac.gpolab.bbn.com/proto-ch/ticket/6),  created by ahelsing on 04-13-2012 at 14:24, last modified: 12-05-2013 at 13:16_

**Comment by tcmitchell** [comment]
This should be OBE with the new CH.

_Trac comment by tmitchel (github user: tcmitchell) on 08-29-2013 at 11:21_

**Comment by ahelsing** [comment]
We'll need to take care of this when we switch the portal to omni 2.0. Here are two alternatives, and there may be others:
 * make omni more flexible about creating the cache dir (possibly disabling the cache if the directory doesn't exist and can't be created)
 * create `/var/www/.gcf` at install time with appropriate permissions

_tmitchel 9/6/12_

**Comment by ahelsing** [comment]
Note that the path to the cache dir is an option.
But your other options are good too.

_ahelsing 9/6/12_

**Comment by ahelsing** [comment]
We use 2.0 now. The cache is in /tmp.
Verify this if fully fixed.

_ahelsing 10/30/12_

**Comment by ahelsing** [comment]
Fixed in late September, changeset ce2af388e8ca00cac65b078f434f107dd1c220eb

_tmitchel 11/2/12_
